### PR TITLE
Updated german translations

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

**Comment by stefonarch** [comment]
Most are comments that not are shown neither in lxqt-applications-menu nor in "menu://applications/" in pcmanfm-qt

File: menu/desktop-directories/lxqt-menu-system.directory.in [hunk 1]
```diff
@@ -100,8 +100,7 @@ Comment[ca]=Preferències personals i dels programes d'administració
 Comment[cs]=Osobní volby a nastavení správy
 Comment[cy]=Hoffterau personol a gosodiadau gweinyddol
 Comment[da]=Personlige indstillinger og administrationskonfiguration
-Comment[de]=Persönliche Anpassungen und Administratoreinstellungen
-Comment[dz]=རང་དོན་དགའ་གདམ་དང་ བདག་སྐྱོང་གི་གཞི་སྒྲིག་ཚུ།
```
Comment (yan12125) [inline menu/desktop-directories/lxqt-menu-system.directory.in:104]: dz translation accidentally dropped? That string is still in https://github.com/lxde/lxmenu-data/blob/991f14ba66f021fea8eaad023da10a04acf0b4a3/po/dz.po#L93

**Review by yan12125 — COMMENTED** [review]
Thanks, looks good overall. Just a small issue.

File: menu/desktop-directories/lxqt-menu-system.directory.in (hunk 1 above)
Comment (stefonarch) [reply menu/desktop-directories/lxqt-menu-system.directory.in:104]: I thought I had fixed that mistake...

**Comment by yan12125** [comment]
No problem, thanks!

**Pull request merged**